### PR TITLE
remote_settings: add email configuration (bug 1898324)

### PR DESCRIPTION
**Pull request by zzzeid**
- add settings for SMTP configuration in remote environments
- set default console backend for local environment

### CODE REVIEW

**Comment by zzzeid** [comment]
Also see https://github.com/mozilla-it/webservices-infra/pull/3622.

File: src/lando/remote_settings.py [hunk 1]
```diff
@@ -39,3 +39,12 @@
     "root": {"handlers": ["null"]},
     "disable_existing_loggers": True,
 }
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
```
Comment (cgsheeh) [inline src/lando/remote_settings.py:43]: `EMAIL_BACKEND` is defined here and in `settings.py`. For consistency, you could remove one, or import to the required file if it should be defined in both places.

**Review by cgsheeh — APPROVED** [review]

File: src/lando/remote_settings.py (hunk 1 above)
Comment (zzzeid) [reply src/lando/remote_settings.py:43]: This is an overridden from the value in `settings`, which is set to the console backend.

Comment (cgsheeh) [reply src/lando/remote_settings.py:43]: Make sense, I didn't notice they are different. Not great that they have the same name and almost the exact same namespace, tbh.

**Review by shtrom — APPROVED** [review]
LGTM.

**Pull request merged**